src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java

```diff
@@ -51,15 +52,14 @@ public List<MappingTableModel> fetchReportList() throws DatabaseReadException {
             throw new DatabaseReadException("No results returned from mapping table");
         }
 
-        log.debug("Result list, a list of objects each representing a row in the mapping table: {}", resultList);
```

Comment:
From a security point of view Is it required to mention table name in the info logs?

Reply:
the name of the table is CSV_TO_SQL_MAPPING table, so we havent logged the table name?

Reply:
Looks good then